cloudant-client/src/test/java/com/cloudant/tests/util/MockWebServerResources.java

```diff
@@ -37,7 +38,37 @@
     public static final TimeUnit TIMEOUT_UNIT = TimeUnit.SECONDS;
 
     public static final String EXPECTED_OK_COOKIE =
-            "AuthSession=\"a2ltc3RlYmVsOjUxMzRBQTUzOtiY2_IDUIdsTJEVNEjObAbyhrgz\"";
+            "a2ltc3RlYmVsOjUxMzRBQTUzOtiY2_IDUIdsTJEVNEjObAbyhrgz";
+    public static final String EXPECTED_OK_COOKIE_2 =
+            "dG9tYmxlbmNoOjU5NTM0QzgyOhqHa60IlqPmGR8vTVIK-tzhopMR";
+
+    // these IAM tokens are treated as opaque strings by the interceptor so it doesn't matter if
+    // they're not representative of the real thing which is a json object with an expiry, etc
+    public static final String IAM_TOKEN = "{\"access_token\": \"IamToken1\"}";
```

Comment:
We should probably mock out the rest of the token content here, given that we know what it looks like.